x-pack/plugins/observability_onboarding/public/routes/custom_logs/index.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+import {
+  CustomLogsSteps,
+  useWizard,
+} from '../../components/app/custom_logs/wizard';
+import { ConfigureLogs } from '../../components/app/custom_logs/wizard/configure_logs';
+import { Inspect } from '../../components/app/custom_logs/wizard/inspect';
+import { InstallElasticAgent } from '../../components/app/custom_logs/wizard/install_elastic_agent';
+import { SelectLogs } from '../../components/app/custom_logs/wizard/select_logs';
+
+function Page({
+  step,
+  component,
+}: {
+  step: CustomLogsSteps;
+  component: React.ReactNode;
+}) {
+  const { setCurrentStep } = useWizard();
+  useEffect(() => {
+    setCurrentStep(step);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [step]);
+
+  return <>{component}</>;
+}
+
+export const customLogsRoutes = {
```

Comment:
this seems like it can be generated in createWizardContext

Reply:
Addressed in [27f765c](https://github.com/elastic/kibana/pull/161230/commits/27f765c86c5254d17f8804d6fb6fb679afd6d00f).